### PR TITLE
Genesis generator as camino-node tool

### PR DESCRIPTION
**Pull request by pnowosie**
# Genesis generator

This tool is used to generate genesis JSON file for the network.

**USAGE:**

```bash
genesis_generator \
  PATH_TO_XLSX_FILE \
  PATH_TO_JSON_TEMPLATE \
  NETWORK_NAME \
  OPTIONAL_OUTPUT_PATH
```
where 
- `NETWORK_NAME` has to be one of: `kopernikus`, `columbus` or `camino`.
- `OUTPUT_PATH` should be a path to the directory where the genesis file will be saved. If ommited, `../genesis/generated` will be used. 

Tool produces a file `genesis_<NETWORK_NAME>.json` in the `OUTPUT_PATH` directory.
<br/>**:warning: Tool was not tested with Windows paths.**

Tool assumes multisignature definitions and allocations are provided in the Excel file. Any other information that shall be contained in the resulting genesis file must be provided in the JSON template.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -141,4 +147,4 @@ require (
 
 replace github.com/ava-labs/coreth => ./dependencies/caminoethvm
 
-replace github.com/ava-labs/avalanchego => ./dependencies/caminoethvm/caminogo
+replace github.com/ava-labs/avalanchego => ../caminogo
```
Comment (c4t-ag) [inline go.mod]: Please change

File: tools/genesis_generator/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+genesis*.json
```
Comment (c4t-ag) [inline tools/genesis_generator/.gitignore]: They should not be ignored

**Review by c4t-ag — CHANGES_REQUESTED** [review]
Please fix the comments, and resolve all the linter issues, thx!

File: tools/genesis_generator/.gitignore (hunk 2 above)
Comment (pnowosie) [reply tools/genesis_generator/.gitignore]: These are results of the tool execution (generated genesis json) this folder isn't a place for these files imo

Comment (c4t-ag) [reply tools/genesis_generator/.gitignore]: But why do you place files in there? The correct folder is "generated" and they should be gitted

**Review by c4t-ag — CHANGES_REQUESTED** [review]
There is one change left (.gitignore) which is IMO not required

**Review by c4t-ag — APPROVED** [review]
Nice one, Thx!

File: tools/genesis_generator/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Genesis generator
+
+This tool is used to generate genesis JSON file for the network.
+
+**USAGE:**
+
+```bash
+genesie_generator \
```
Comment (pnowosie) [inline tools/genesis_generator/README.md]:
```suggestion
genesis_generator \
```

**Pull request merged**